frontend/src/libs/components/text-editor/libs/extensions/font-size/libs/helpers/increment-font-size.helper.ts

```diff
@@ -0,0 +1,10 @@
+import { FontSizeConfig } from '../constants/config.constant.js';
+
+const incrementFontSize = (fontSize: number): number => {
+  const newFontSize = fontSize + FontSizeConfig.CHANGE_STEP;
+  return newFontSize > FontSizeConfig.MAX_FONT_SIZE
+    ? FontSizeConfig.MAX_FONT_SIZE
+    : newFontSize;
```

Comment:
I believe the same, but more compact
```suggestion
  return fontSize === FontSizeConfig.MAX_FONT_SIZE
    ? fontSize
    : fontSize + FontSizeConfig.CHANGE_STEP;
```

Reply:
But what if starting font size is an odd number ? Let's say 21px.  So it will go to 23 on first increment and then to 25 on second ( and our limit is 24 )

Reply:
Okay 🤔